test/Trip-test.js

```diff
@@ -0,0 +1,104 @@
+import { expect } from 'chai';
+import Trip from '../src/Trip';
+import tripData from '../test-data/trip-data';
+import destinationData from '../test-data/destination-data';
+
+describe('Trip', () => {
+  let trip1;
+  let testDestination1Data;
+
+  beforeEach(() => {
+    trip1 = new Trip(tripData);
+    testDestination1Data = destinationData;
+  });
+
+  it('should be a function', () => {
+    expect(Trip).to.be.a('function');
+  });
+
+  it('should be an instance of Trip', () => {
+    expect(trip1).to.be.an.instanceof(Trip);
+  });
+
+  it('should be able to take in trip data', () => {
+    expect(trip1.data).to.be.an('array');
+    expect(trip1.data).to.deep.equal(tripData);
+
+  });
+
+  it('should be able to test whether the traveler id is valid and also return a message if it\'s not valid', () => {
```

Comment:
great error handling- i hadn't thought of this. will be great to have this for the login, page too! 🐲